src/app/frontend/common/components/date/template.html

```diff
@@ -14,11 +14,7 @@
 limitations under the License.
 -->
 
-<span *ngIf="_relative"
-      class="kd-date"
+<span class="kd-date"
       [matTooltip]="date | date:format">
-  {{date | kdRelativeTime}}
-</span>
-<span *ngIf="!_relative">
-  {{date | date:format}}
+  {{_relative ? (date | kdRelativeTime) : (date | date)}}
```

Comment:
Shouldn't it be `date:format` at the end?

Reply:
`date:format` gives the longer version of the timestamp which is sometimes longer than column displays. Since the longer version is already shown in the tooltip, I think the main display can be the shorter one with `date`.